test-cases/longevity/longevity-large-partition-4days.yaml

```diff
@@ -2,8 +2,14 @@ test_duration: 6480
 
 bench_run: true
 
-prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -max-rate=300 -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -concurrency=7 -rows-per-request=10"]
-stress_cmd: ["scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -rows-per-request=10 -concurrency=7 -max-rate=32000 -duration=5760m"]
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=20 -clustering-row-count=10000000 -clustering-row-size=2048 -concurrency=50000 -rows-per-request=2000 -timeout=180s -connection-count 10000",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=20 -clustering-row-count=10000000 -partition-offset=20 -clustering-row-size=2048 -concurrency=50000 -rows-per-request=2000 -timeout=180s -connection-count 10000",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=20 -clustering-row-count=10000000 -partition-offset=40 -clustering-row-size=2048 -concurrency=50000 -rows-per-request=2000 -timeout=180s -connection-count 10000"
+                    ]
+stress_cmd: ["scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=60 -clustering-row-count=10000000 -clustering-row-size=2048 -rows-per-request=2000 -timeout=180s -concurrency=10000 -max-rate=32000 -duration=5760m",
```

Comment:
Why do you use "-partition-count=60" if we have in total 60 partitions  count?
Is it choosing randomly 1  out of these 60? (I recall something like that, but not sure).

Reply:
You didn't add here "-connection-count  1000"

Reply:
there are 60 partitions and each one of the reads uses 'uniform' to choose any partition and scan it. No issues are expected by that. In addtion, s-b doesn't support a partition offset in this case.

regarding "-connection-count 1000", i wasn't sure how it works with the read stress. will test it as well.